home/forms.py

```diff
@@ -5,6 +5,12 @@
 from home.models import Point, WgUpdate, Minutes
 from users.models import Profile
 
+def validate_uoe_mail(value):
```

Comment:
`is_university_email` or similar might be more intuitive naming, e.g. leads to `if not is_university_email() ...`.